etc/genome/spec/sys_lock_exit_code.yaml

```diff
@@ -0,0 +1,3 @@
+---
+env: XGENOME_SYS_LOCK_EXIT_CODE
+default_value: ' '
```

Comment:
But 0 is a valid exit code value.  You'd have to use -1 or 65536 (something greater than you can store in an int), or something